File: src/main/java/com/adjorno/billib/rest/db/GlobalRankArtist.java
package com.adjorno.billib.rest.db;

import javax.persistence.*;

@Entity
@Table(name = "GLOBAL_RANK_ARTIST")
public class GlobalRankArtist {
    @Id
    @Column(name = "RANK")
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long mRank;

    @Column(name = "ARTIST_ID")
    private Long mArtistId;

    public GlobalRankArtist() {
    }

    public Long getRank() {
        return mRank;
    }

    public void setRank(Long rank) {
        mRank = rank;
    }

    public Long getArtistId() {
        return mArtistId;
    }

    public void setArtistId(Long artistId) {
        mArtistId = artistId;
    }
}

File: src/main/java/com/adjorno/billib/rest/db/GlobalRankArtist.kt
package com.adjorno.billib.rest.db

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "GLOBAL_RANK_ARTIST")
data class GlobalRankArtist(
    @Id
    @Column(name = "_RANK")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var rank: Long? = null,

    @Column(name = "ARTIST_ID")
    var artistId: Long? = null
)
